compose-dash.sh

```diff
@@ -52,14 +52,14 @@ else
     pwextend=""
 fi
 
-echo "Running Docker Compose..."  
+echo "Running Docker Compose..."
 if docker-compose version > /dev/null 2>&1; then
     # Build Docker (v1)
-    docker-compose -f powerwall.yml $pwextend $1 $2
+    docker-compose -f powerwall.yml $pwextend $1 $2 $3
```

Comment:
Thanks @mcbirse  - This makes sense. In fact, had me wonder if we should just include all arguments. Should we just use `$@` which would be all arguments?

Reply:
Hi Jason, I agree, it would probably make sense to simply use `$@` so there is no limit on the number of additional arguments that could be passed. Please feel free to update it.